api/envoy/api/v3alpha/cluster/circuit_breaker.proto

```diff
@@ -5,8 +5,6 @@ package envoy.api.v3alpha.cluster;
 option java_outer_classname = "CircuitBreakerProto";
 option java_multiple_files = true;
 option java_package = "io.envoyproxy.envoy.api.v3alpha.cluster";
-option csharp_namespace = "Envoy.Api.V2.ClusterNS";
```

Comment:
These don't need to be in v3alpha, they will go away after we re-org `envoy.api.v3alpha`.